redash/handlers/admin.py

```diff
@@ -23,6 +25,13 @@ def outdated_queries():
     else:
         outdated_queries = []
 
+    record_event({
+        'action': 'view',
+        'object_type': 'api_call',
+        'object_id': 'admin/outdated_queries',
+        'timestamp': int(time.time()),
+    })
```

Comment:
`action` -> `list`
`object_type` -> `outdated_queries`

(`object_id` can be null and can be ommited)